<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrajEdtDlg</class>
 <widget class="QDialog" name="TrajEdtDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>131</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>trajectory point value</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1,0">
     <item>
      <layout class="QGridLayout" name="gridLayout" rowstretch="0,0">
       <item row="1" column="1">
        <widget class="QComboBox" name="mCmbBikeType"/>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="mSpinID"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>ID</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Type</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>50</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,0">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="mBtSave">
         <property name="text">
          <string>save</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="mBtCancel">
         <property name="text">
          <string>cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
